nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -765,65 +805,51 @@ class ReconAll(CommandLine):
             ('fill', ['mri/filled.mgz',
                       # 'scripts/ponscc.cut.log',
                       ], []),
-            ('tessellate', ['surf/lh.orig.nofix', 'surf/rh.orig.nofix'], []),
-            ('smooth1', ['surf/lh.smoothwm.nofix', 'surf/rh.smoothwm.nofix'],
-             []),
-            ('inflate1', ['surf/lh.inflated.nofix', 'surf/rh.inflated.nofix'],
-             []),
-            ('qsphere', ['surf/lh.qsphere.nofix', 'surf/rh.qsphere.nofix'],
-             []),
-            ('fix', ['surf/lh.orig', 'surf/rh.orig'], []),
-            ('white', ['surf/lh.white', 'surf/rh.white',
-                       'surf/lh.curv', 'surf/rh.curv',
-                       'surf/lh.area', 'surf/rh.area',
-                       'label/lh.cortex.label', 'label/rh.cortex.label'], []),
-            ('smooth2', ['surf/lh.smoothwm', 'surf/rh.smoothwm'], []),
-            ('inflate2', ['surf/lh.inflated', 'surf/rh.inflated',
-                          'surf/lh.sulc', 'surf/rh.sulc',
-                          'surf/lh.inflated.H', 'surf/rh.inflated.H',
-                          'surf/lh.inflated.K', 'surf/rh.inflated.K'], []),
+            ]
+        _autorecon2_lh_steps = [
```

Comment:
Because I specify `lh` in the filenames. I then make a `rh` copy below, just swapping the filenames out.